dynamic/README.md

```diff
@@ -243,6 +243,28 @@ func ReadResourceRequest(i *tfprotov6.ReadResourceRequest) *tfplugin6.ReadResour
 	}
 }
 ```
+
+### `package parameterize`
```

Comment:
Thanks for documenting! Docs are very good to have. 

We're still figuring out what's our style for docs - so happy to discuss alternatives. One pattern I've used before in this repo is `package.go` with doc comment describing what each package does. This seems to be some support for this in e.g. https://tip.golang.org/doc/comment#package - this also gets copied into auto-generated Go docs.


The README is also very important. There's "how do I use this" and "what can it do" concerns that go in a README. Not sure what's the right balance for us is here but my 2c so you're aware of the package.go option for the package-level detail.

Reply:
It's a good call-out. The bridge also has 1 `doc.go`, and I've seen more in the wild.

For this PR, I'm going to keep using the README.md, since that's what the module already does. It might be worth standardizing at some point.

Autogenerated Go docs also include the README: https://pkg.go.dev/github.com/pulumi/pulumi-terraform-bridge/dynamic.